Corona-Warn-App/src/main/assets/privacy_de.html

```diff
@@ -199,7 +199,7 @@ <h2>
     Smartphone nicht über diese Zufalls-IDs identifiziert werden kann. Die von
     Ihrem Smartphone ausgesendeten eigenen Begegnungsdaten und die
     aufgezeichneten Begegnungsdaten der Personen, mit denen Sie Kontakt hatten,
-    werden auf Ihrem Smartphone gespeichert und jeweils nach 14 Tagen gelöscht.
+    werden auf Ihrem Smartphone gespeichert und jeweils nach 14 Tage gelöscht.
```

Comment:
@janetback I used the html file provided by the lawyers but I think that's wrong...

Reply:
Hi @harambasicluka, hier sollte es tatsächlich 'Tagen' heißen.